### DISCUSSION TITLE
table inside a EditPage

### DISCUSSION THREAD

**Discussion by carlituxman** [filamentphp/filament#570] [Help] [answered]
Is possible inside a EditPage have a relation table. For example editing a Customer resource show a tab with a Orders table from Customer

How can I do it?

**Comment by danharrin** [comment, marked as answer]
You need to use a relation manager, theres a section in the docs :)

**Reply by carlituxman** [reply]
Si possible use some diferents in some tabs??

**Reply by danharrin** [reply]
> Si possible use some diferents in some tabs??

Not yet unfortunately
